sass/components/_on-this-page.scss

```diff
@@ -12,29 +11,24 @@
     padding-inline-start: $h-padding;
   }
 
-  >ul {
+  > ul {
     padding-inline: $h-padding;
   }
 
-  li[data-active=true] {
-    font-weight: bolder;
-
-    li {
-      font-weight: normal;
-    }
+  li[data-active="true"] > a {
+    color: #ececec;
   }
 
   a {
     display: block;
     text-wrap: balance;
     padding-block: 3px;
-    // copied from .media-content
+    color: #868686;
     text-decoration: none;
-    color: $link-color;
     word-break: break-word;
 
     &:hover {
-      text-shadow: 0 0 0.9px $link-hover-shadow-color, 0 0 0.9px $link-hover-shadow-color;
+      color: lighten(#868686, 20);
```

Comment:
I'm not entirely sure if there's an existing color that would work for this (a color between the gray and white), so I just used `lighten`. Let me know if there's a better fit

Reply:
I've tried it locally and for me it looks perfect, subtle yet visible. 👍